src/screens/HomeScreen.tsx

```diff
@@ -50,6 +50,7 @@ const styles = StyleSheet.create({
     justifyContent: 'space-evenly',
     alignItems: 'center',
     height: '100%',
+    paddingBottom: '10%',
```

Comment:
Waarvoor is dit?

Reply:
Dat was toen voor de buttons van de homepage dat ze spacing ertussen hebben, maar dat is al gefixt in de homescreen pr